src/dosext/mfs/mfs.h

```diff
@@ -42,8 +42,8 @@ typedef struct vm86_regs state_t;
 
 #define MASK8(x)	((x) & 0xff)
 #define MASK16(x)	((x) & 0xffff)
-#define HIGH(x)		MASK8((unsigned long)(x) >> 8)
-#define LOW(x)		MASK8((unsigned long)(x))
+#define HIGH(x)		(uint8_t)MASK8((unsigned long)(x) >> 8)
```

Comment:
What kind of printf and what
casting? Maybe the casting to
`long` should go away instead?

Reply:
Just did `printf("%d\n", HIGH(eax));` and got warning about needing `%lu` format.

Reply:
Incidentally shouldn't `LOW` and `HIGH` be called `LBYTE` and `HBYTE`?

Reply:
I think we already have the macros
`LO_BYTE()` and alike.
Anyway, have you tried to remove
the cast to long instead?

Reply:
> Anyway, have you tried to remove
the cast to long instead?

Not yet, but will do.

Reply:
Seems to work fine, thanks.